apis/discovery/v1alpha1/foreigncluster_types.go

```diff
@@ -125,6 +125,9 @@ type Incoming struct {
 // +kubebuilder:resource:scope=Cluster
 
 // ForeignCluster is the Schema for the foreignclusters API
+// +kubebuilder:printcolumn:name="Outgoing joined",type=string,JSONPath=`.status.outgoing.joined`
+// +kubebuilder:printcolumn:name="Incoming joined",type=string,JSONPath=`.status.incoming.joined`
+// +kubebuilder:printcolumn:name="Auth status",type=string,JSONPath=`.status.authStatus`
```

Comment:
What do you think about using "Authorization status"?

Reply:
Sorry @lomluca, the name should be "Authentication status", instead of "Authorization status". My fault :disappointed:

Reply:
No worries, done!